lib/config/folderconfiguration.go

```diff
@@ -8,19 +8,17 @@ package config
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
 	"runtime"
-	"strings"
 
-	"github.com/syncthing/syncthing/lib/osutil"
+	"github.com/syncthing/syncthing/lib/fs"
 	"github.com/syncthing/syncthing/lib/protocol"
 )
 
 type FolderConfiguration struct {
 	ID                    string                      `xml:"id,attr" json:"id"`
 	Label                 string                      `xml:"label,attr" json:"label"`
-	RawPath               string                      `xml:"path,attr" json:"path"`
+	FilesystemType        string                      `xml:"filesystemType,attr" json:"filesystemType"`
```

Comment:
Minor style nit, maybe these should not be `attr`s as it's getting a bit long, but no strong opinion really.